sdk/appconfiguration/azure-appconfiguration-provider/azure/appconfiguration/provider/_azureappconfigurationprovider.py

```diff
@@ -66,6 +67,9 @@ def load(
     *,
     selects: Optional[List[SettingSelector]] = None,
     trim_prefixes: Optional[List[str]] = None,
+    keyvault_credential: Optional["TokenCredential"] = None,
+    keyvault_client_configs: Optional[Mapping[str, JSON]] = None,
+    secret_resolver: Optional[Callable[[str], str]] = None,
```

Comment:
We have an incoming pylint rule that states that all documented parameters/keywords need to be in the signature so I added them here. This also clears up a lot of the invalid overload for `load` errors you were seeing

Reply:
Thanks, those must have been missed while we were adding things, they should have been here.